src/analysis/multiSpecies/microbiomeModelingToolbox/mgPipe/microbiotaModelSimulator.m

```diff
@@ -77,7 +77,7 @@
 % Starting personalized simulations
 for k = startIter:(patNumb + 1)
  idInfo = cell2mat(sampName((k - 1), 1))
- load(strcat('microbiota_model_samp_', idInfo, '.mat'))
+ load(strcat('microbiota_model_samp_', idInfo, '.mat'));
```

Comment:
redCbModel ? Or is there a specific reason for load?

Reply:
Hi Thomas,
I tried to use readCbModel but I got errors. I think it is because the structure of the microbiota models is a bit different from the usual...  (additional fields for example )

Reply:
Additional fields should not be an issue.
The problem is that this is then likely to fail on anything but exactly those models, and can lead to problems in the future, if things change for whatever reason. 
I'll have a look at this and come back to you.

Reply:
Thanks for your help Thomas :)

Reply:
Do you have some script with supplemental  files that I could use to test this?
Some kind of test that can be run on the toolbox (potentially with a smaller subset of models) that reproduces your issues with readCbModel ?
We could base a test of mgPipe on that test once we have sorted out the problems.

Reply:
Hi Thomas,
https://github.com/opencobra/COBRA.tutorials/tree/develop/additionalTutorials/microbiomeModelingToolbox
Here, in the tutorial,  you have all the inputs (except then the agora models to be downloaded from vmh.life)  and the steps to reproduce the all Pipeline for a really small example (it should take around 10 minutes).
Otherwise, if you prefer to have directly the models let me know I can try to send them to you....
thanks
fede

Reply:
running mgPipeTutorial leads to errors:
1. solver is not defined in the driver (i set it by `solver = 'ibm_cplex'`)
2. the initMgPipe call does not work:
```
Error using initMgPipe
Too many input arguments.

Error in mgPipeTutorial (line 93)
[init,modPath,toolboxPath,resPath,dietFilePath,abunFilePath,objre,figForm,solver,numWorkers,autoFix,compMod,patStat,rDiet,extSolve,fvaType,autorun]= initMgPipe(modPath, toolboxPath, resPath, dietFilePath, abunFilePath, objre, figForm, solver, numWorkers, autoFix, compMod, patStat, rDiet,extSolve,fvaType,autorun);
```
And admittedly: This call is way to long (isn't there any way to provide defaults of parameter value pairs)? its hard to say what is "wrong".

Reply:
Hi Thomas,
Yes, you are right. The 'solver' input got deprecated as it is set automatically by initCobra toolbox. 
I guess the problem with the tutorial is that am guilty of having forgotten to remove the deprecated 'solver' parameter from the call to the initMgPipe function. I will fix this. If you still want to try to run it you can easily fix it by removing solver from the call of the function. Everything else should work. Sorry for the inconvenience. 

Concerning the number of parameters, I am not sure...

Reply:
Ok...
This is very likely an issue with coupling constraints.